npm/Dockerfile

```
@@ -0,0 +1,27 @@
+# This file is automatically generated by the Nebula SDK. DO NOT EDIT.
+
+FROM node:12
```

I'm going to try using `lts-alpine`

```suggestion
FROM node:lts-alpine
```

---

Although then you can't use apt-get and may need to copy something that uses apk like https://github.com/puppetlabs/nebula-steps/blob/master/helm/Dockerfile